File: myExpenses/src/androidTest/java/org/totschnig/myexpenses/test/provider/CurrencyInfo.java
package org.totschnig.myexpenses.test.provider;

import android.content.ContentValues;

import org.totschnig.myexpenses.provider.DatabaseConstants;

public class CurrencyInfo {
  String label;
  String code;

  public CurrencyInfo(String label, String code) {
    this.label = label;
    this.code = code;
  }
  public ContentValues getContentValues() {
    ContentValues v = new ContentValues();

    v.put(DatabaseConstants.KEY_LABEL, label);
    v.put(DatabaseConstants.KEY_CODE, code);
    return v;
  }

}

File: myExpenses/src/androidTest/java/org/totschnig/myexpenses/testutils/CurrencyInfo.kt
package org.totschnig.myexpenses.testutils

import android.content.ContentValues
import org.totschnig.myexpenses.provider.KEY_CODE
import org.totschnig.myexpenses.provider.KEY_LABEL

class CurrencyInfo(val label: String, val code: String) {
    val contentValues = ContentValues().apply {
        put(KEY_LABEL, label)
        put(KEY_CODE, code)
    }
}
